### PR TITLE
create general python function

### PR DESCRIPTION
**Pull request by thiagoeffarias**
This pr focus on creating general math functions for the other future files.

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def sumInt(a, b):
+    return a + b
```
Comment (blapolli) [inline functions.py]: ## Please, use print(a+b)

**Review by blapolli — COMMENTED** [review]
Good job Thiago, you are on the right track!

File: functions.py (hunk 1 above)
Comment (thiagoeffarias) [reply functions.py]: Ok, i gonna change it. Thanks.

Comment (blapolli) [reply functions.py]: Thank you jerk!

**Review by blapolli — APPROVED** [review]
LGTM 🥇

**Pull request merged**